test/addurl_test.go

```diff
@@ -69,8 +69,10 @@ TAG ` + tag
 	})
 	defer os.RemoveAll(buildDir)
 
-	err := runRockerBuildWdWithOptions(buildDir, "-cache-dir", cacheDir)
-	assert.Nil(t, err, "no difference in downloaded and added files")
+	err := runRockerBuildWd(buildDir, "-cache-dir", cacheDir)
+	if err != nil {
```

Comment:
Please, read what I wrote. You can use `assert.Nil` of `assert.NoError` as a parameter to `if`, because it will return a nil.